tests/python_package_test/test_dask.py

```diff
@@ -57,7 +57,8 @@
 pytestmark = [
     pytest.mark.skipif(getenv('TASK', '') == 'mpi', reason='Fails to run with MPI interface'),
     pytest.mark.skipif(getenv('TASK', '') == 'gpu', reason='Fails to run with GPU interface'),
-    pytest.mark.skipif(getenv('TASK', '') == 'cuda_exp', reason='Fails to run with CUDA Experimental interface')
+    pytest.mark.skipif(getenv('TASK', '') == 'cuda_exp', reason='Fails to run with CUDA Experimental interface'),
+    pytest.mark.skipif(getenv('TASK', '') == 'int64', reason='Fails to run with Dataset rows with int64 interface')
```

Comment:
Can you please explain *specifically* which Dask tests failed as a result of the changes in this PR? The Dask tests are the most thorough tests of distributed training in this repo, so I'm very nervous about skipping them on a PR that makes such invasive changes.

Reply:
Thank you. 
I enabled the dask tests for int64